scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -444,6 +446,7 @@ <h6>No causative variants</h6>
               </div>
               <div class="row">
                 <div class="col-4">
+                  <i class="far fa-check-circle" style="color:transparent; background-color:transparent"></i>
```

Comment:
This is not a very good solution but I had trouble indenting the text to align with the text in previous rows in the table in a reliable way.